### PR TITLE
BUGFIX: focus content node on creation

### PR DESCRIPTION
**Pull request by dimaip**
Fixes: #939

### CODE REVIEW

**Comment by dimaip** [comment]
@mstruebing that's a good point, I've thought about it too. In the old UI it does do that. It's kind of hard to implement. If a node has multiple editables, which one should it focus?
In any case I propose to do that in an additional PR, maybe after 1.0. I'm trying hard not to go over-budget with all of these tiny bugfixes, so if it works the way as it did in the old UI IMO it's already good enough for now.

File: packages/neos-ui/src/manifest.js [hunk 1]
```diff
@@ -270,6 +270,7 @@ manifest('main', {}, globalRegistry => {
         const contentElement = (new DOMParser())
             .parseFromString(renderedContent, 'text/html')
             .querySelector(`[data-__neos-node-contextpath="${contextPath}"]`);
+        const fusionPath = contentElement.dataset.__neosFusionPath;
```
Comment (skurfuerst) [inline packages/neos-ui/src/manifest.js]: contentElement might be NULL, see check 3 lines below. So move this after this ;)

**Comment by dimaip** [comment]
@skurfuerst Oh gosh, good catch! Fixed.

**Comment by dimaip** [comment]
Merging, since the only review comment is now fixed.

**Pull request merged**